hw/ip_examples/im2col_spc/data/im2col_spc.hjson

```diff
@@ -87,15 +87,15 @@
       swaccess: "rw",
       hwaccess: "hro",
       fields: [
-        { bits: "7:0", name: "NUM", desc: "Number of patches" }
+        { bits: "15:0", name: "NUM", desc: "Number of patches" }
```

Comment:
no this could have been left to 8 bits, I just meant the 32bit numbers that should be 16bit

Reply:
unless this is the change we made yesterday on the address part :)

Reply:
All right! Btw, do you think that the input image might be bigger than 256x256? If so, 8 bits are not enough

Reply:
I think we can indeed make it 16bits